collector/filesystem_bsd.go

```diff
@@ -33,6 +33,7 @@ import "C"
 
 const (
 	defIgnoredMountPoints = "^/(dev)($|/)"
+	MNT_RDONLY            = 0x1
```

Comment:
The os package is known to violate several go idioms as it's very old. Go core contributors would like to rewrite it if they could